### DISCUSSION TITLE
预渲染无法正常使用

### DISCUSSION THREAD

**Discussion by cmdparkour** [NervJS/taro#10976] [Feature Request]
建议官方针对预渲染的各种异常情况，出一下排查方法的思路。
毕竟，一般使用预渲染都是在项目达到一定规模的时候才会使用，但往往这时候开启预渲染，即使是新建路由，也会出现预渲染能力无法正常使用的情况，按理说，应该会直接生成对应的文件才对。

**Comment by Chen-jj** [comment]
3.0 应该没问题

3.1-3.3 只要使用了 Taro API 时会报错，3.4 会修复 #8959

另外就是使用了一些 Taro API 时会报错，这点最近已在文档中进行补充：[使用了部分 Taro API 后预渲染报错​](https://docs.taro.zone/docs/prerender#%E4%BD%BF%E7%94%A8%E4%BA%86%E9%83%A8%E5%88%86-taro-api-%E5%90%8E%E9%A2%84%E6%B8%B2%E6%9F%93%E6%8A%A5%E9%94%99)

相关 Issues：[is:issue prerender is:closed ](https://github.com/NervJS/taro/issues?q=is%3Aissue+prerender+is%3Aclosed)
